Source/artifacts/QueryforInquirer.js

```diff
@@ -0,0 +1,106 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Folders } from '../Folders';
+import fs from 'fs';
+import global from '../global'
+
+const inquirer = require('inquirer');
+
+const _folders = new WeakMap();
+const _fileSystem = new WeakMap();
+
+export class QueryforInquirer {
+
+    /**
+     * Constructor
+     * @param {Folders} folders
+     * @param {fs} fileSystem
+     */
+    constructor(folders, fileSystem) {
+        _folders.set(this, folders);
+        _fileSystem.set(this, fileSystem);
+    }
+    /**
+     * Gets the inquirer.js prompt answers based on the language
+     * @param {String} language
+     * @returns {Promise<any>} The answers
+     */
+    promptUser(language){
+        if (language === 'csharp') {
+            return this._getCSharpPrompt()
+        }
+    }
+
+    _getCSharpPrompt() {
+        const customReadModel = 'Write read model name';
+        const namespace = global.createCSharpNamespace(process.cwd(), global.getNearestCsprojFile());
+        console.log(namespace);
+
+        let readModelChoices = this._findCSharpReadmodels();
+        readModelChoices.push(customReadModel);
+        let questions = [
+            {
+                type: 'input',
+                name: 'name',
+                message: `What's the Query's name?`
+            },
+            {
+                type: 'rawlist',
+                name: 'readModel',
+                message: 'Choose a read model: ',
+                choices: readModelChoices
+            },
+            {
+                type: 'input',
+                name: 'readModelCustom',
+                message: 'Write the name of the read model: ',
+                when: function(answers) {
+                    return answers.readModel === customReadModel;
+                }
+            },
+            {
```

Comment:
An example of something needing to be dealt with for #34. We should be able to feed language specific queries from a different place. Namespace is C# specific and should be somewhere else.